tests/base.py

```diff
@@ -161,10 +161,6 @@ def expected_metadata(self):
                 self.PRIMARY_KEYS: {'id'},
                 self.REPLICATION_METHOD: self.FULL
             },
-            "cash_drawer_shifts": {
-                self.PRIMARY_KEYS: {'id'},
-                self.REPLICATION_METHOD: self.FULL
-            },
```

Comment:
This is duplicate code that probably got merged in when there was a conflict, same lines exist higher up in the map